src/main/java/com/marklogic/batch/rdbms/Main.java

```diff
@@ -0,0 +1,41 @@
+package com.marklogic.batch.rdbms;
+
+import com.marklogic.spring.batch.core.launch.support.CommandLineJobRunner;
+import joptsimple.OptionParser;
+
+import java.util.Arrays;
+
+public class Main extends CommandLineJobRunner {
```

Comment:
Can you add class comments to explain that this is the entry point, and its job is to specify the Spring Batch configuration class (probably need to explain what that is) and read in command line options?